server/src/main/java/keywhiz/service/daos/DaoModule.java

```diff
@@ -0,0 +1,9 @@
+package keywhiz.service.daos;
+
+import com.google.inject.AbstractModule;
+
+public class DaoModule extends AbstractModule {
+  @Override protected void configure() {
+    bind(ClientMapper.class).toProvider(ClientMapper::new);
+  }
+}
```

Comment:
ClientMapper is package-private and has an implicit constructor. Rather than increase visibility or add @Inject to the constructor, and to support testing, I created a same-package module that explicitly binds the constructor.